src/main/java/org/jenkinsci/plugins/reverse_proxy_auth/ReverseProxySecurityRealm.java

```diff
@@ -31,12 +31,9 @@
 import hudson.model.Descriptor;
 import hudson.model.Hudson;
 import hudson.model.User;
-import hudson.security.GroupDetails;
-import hudson.security.UserMayOrMayNotExistException;
+import hudson.security.*;
```

Comment:
bad idea for the production code. Intellij IDEA with default settings does it unfiirtunately